ISOLATE_STDLIB_MODULES.rst

```diff
@@ -0,0 +1,206 @@
+PEP: XXX
+Title: Isolating modules in the standard library
+Author: Erlend Egeberg Aasland <erlend.aasland@protonmail.com>, Petr Viktorin <encukou@gmail.com>
+Discussions-To: XXX
+Status: Draft
+Type: Standards
+Requires: 489, 573, 630
+Created: 16-Feb-2022
+Python-Version: 3.11
+Post-History: XXX
+
+
+Abstract
+========
+
+Extensions in the standard library will be converted to multi-phase
+initialization (:pep:`489`) and where possible, all state will be
+stored on module objects rather than in process-global variables.
+
+
+Note on Backdating
+==================
+
+Much of this proposal has already been implemented, partly to solve related
+issues and partly because of misunderstandings and lack of coordination.
+
```

Comment:
One last nitpick: IMO, the important piece of information for the reader is that much of this has already been implemented. There's probably a lot of reasons why (finding all of them would be a time-consuming task), but I'm not sure those reasons are valuable information in this context. Suggesting a simplification:

```suggestion
Much of this proposal has already been implemented.
```